src/rustup-cli/help.rs

```diff
@@ -78,7 +78,23 @@ inferred, so the above could be written:
     $ rustup default stable-msvc
 
 Toolchain names that don't name a channel instead can be used to name
-custom toolchains with the `rustup toolchain link` command.";
+custom toolchains with the `rustup toolchain link` command. This can
+also be used to symlink toolchains from local builds. For more
+information see `rustup toolchain help link`.";
+
+pub static TOOLCHAIN_LINK_HELP: &'static str =
+r"
+Symlinks a toolchain from a local directory specified by 'path' and
+gives it a custom name specified by 'toolchain' if it does not name a
+standard release channel.
```

Comment:
Ah, I see. Then I think the text should be clearer about this. How about:

> Symlinks a toolchain from a local directory specified by 'path' and gives it a custom name specified by 'toolchain'. If `toolchain` matches a standard release channel such as `stable`, `nightly`, `1.8.0`, etc., then `path` will be used in place of the standard implementations of those toolchains. For example, if a toolchain symlink is set up called `nightly`, any directory with a `nightly` override will now also use that toolchain.

Also, I notice the text currently says
> For more information see `rustup help toolchain`
Does `rustup help toolchain` *actually* give more information at the moment?